scripts/integrationtest.sh

```diff
@@ -29,14 +27,9 @@ mkdir $BTC_DIR
 CFG_DIR="$TMP_DIR/cfg"
 mkdir $CFG_DIR
 
-# Build all executables
-cd $SRC_DIR
-cargo build --release
-BIN_DIR="$SRC_DIR/target/release"
-
 # Generate federation, gateway and client config
-$BIN_DIR/configgen -- $CFG_DIR 4 4000 5000 1000 10000 100000 1000000 10000000
```

Comment:
This assumes people have nix. I'd prefer to make that optional, e.g. set the `BIN_DIR` environment variable in the nix shell if it's present it is used otherwise the integration test will build minimint and set `BIN_DIR` itself.

Reply:
I have reset the changes in the script and made necessary changes in nix files to avoid this issue